docs/faq/gce.md

```diff
@@ -0,0 +1,262 @@
+(eg: `be` for backends, `hc` for health checks). If a given resource is not tied
+to a single `node-port`, its name will not include the same.
+
+## Can I change the cluster UID?
+
+The Ingress controller configures itself to add the UID it stores in a configmap in the `kube-system` namespace.
+
+```console
+$ kubectl --namespace=kube-system get configmaps
+NAME          DATA      AGE
+ingress-uid   1         12d
+
+$ kubectl --namespace=kube-system get configmaps -o yaml
+apiVersion: v1
+items:
+- apiVersion: v1
+  data:
+    uid: UID
+  kind: ConfigMap
+...
+```
+
+You can pick a different UID, but this requires you to:
+
+1. Delete existing Ingresses
+2. Edit the configmap using `kubectl edit`
+3. Recreate the same Ingress
+
+After step 3 the Ingress should come up using the new UID as the suffix of all cloud resources. You can't simply change the UID if you have existing Ingresses, because
+renaming a cloud resource requires a delete/create cycle that the Ingress controller does not currently automate. Note that the UID in step 1 might be an empty string,
+if you had a working Ingress before upgrading to Kubernetes 1.3.
+
+__A note on setting the UID__: The Ingress controller uses the token `--` to split a machine generated prefix from the UID itself. If the user supplied UID is found to
+contain `--` the controller will take the token after the last `--`, and use an empty string if it ends with `--`. For example, if you insert `foo--bar` as the UID,
+the controller will assume `bar` is the UID. You can either edit the configmap and set the UID to `bar` to match the controller, or delete existing Ingresses as described
+above, and reset it to a string bereft of `--`.
+
+## Why do I need a default backend?
+
+All GCE URL maps require at least one [default backend](https://cloud.google.com/compute/docs/load-balancing/http/url-map#url_map_simplest_case), which handles all
+requests that don't match a host/path. In Ingress, the default backend is
+optional, since the resource is cross-platform and not all platforms require
+a default backend. If you don't specify one in your yaml, the GCE ingress
+controller will inject the default-http-backend Service that runs in the
+`kube-system` namespace as the default backend for the GCE HTTP lb allocated
+for that Ingress resource.
+
+## How does Ingress work across 2 GCE clusters?
+
+See federation [documentation](http://kubernetes.io/docs/user-guide/federation/federated-ingress/).
+
+## I shutdown a cluster without deleting all Ingresses, how do I manually cleanup?
+
+If you kill a cluster without first deleting Ingresses, the resources will leak.
+If you find yourself in such a situation, you can delete the resources by hand:
+
+1. Navigate to the [cloud console](https://console.cloud.google.com/) and click on the "Networking" tab, then choose "LoadBalancing"
+2. Find the loadbalancer you'd like to delete, it should have a name formatted as: k8s-um-ns-name--UUID
```

Comment:
Would be nice for it to be clear what's a variable and what's not. No clue if "-um-" is literal of meant to stand for something.

e.g. `k8s-???-$namespace-$name--$UUID` is fairly well understood for indicating variables.